cypress/e2e/orders/orders.js

```diff
@@ -73,6 +86,7 @@ describe("Orders", () => {
         createShipping({
           channelId: defaultChannel.id,
           name: randomName,
+          price: shippingPrice,
           address,
           taxClassId: taxClass.id,
         });
```

Comment:
In productsUtlis we have function "createProductWithShipping", maybe we can use it instead of (createShipping, createTypeAttributeAndCategoryForProduct, createProductInChannel)? It will be more clear

Reply:
I promise I will take a look at our data preparation set soon - for now, let us stick with the old one